main.py

```diff
@@ -1,142 +1,129 @@
-import sys
-import pygame
+from src.settings import *
+from src.support import *
+from src.level import Level
+from src.main_menu import MainMenuInternal
+from src.pause_menu import PauseMenu
+from src.settings_menu import SettingsMenu
 
-from src import settings
-from src import support
-from src import level
-from src import main_menu
 
 
-class Game:
+
+class Game():
     def __init__(self):
-        self.character_frames: dict[str, settings.AniFrames] | None = None
-        self.level_frames: dict | None = None
-        self.tmx_maps: settings.MapDict | None = None
-        self.overlay_frames: dict[str, pygame.Surface] | None = None
-        self.font: pygame.font.Font | None = None
-        self.sounds: settings.SoundDict | None = None
+        # pygame
+        self.screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
         pygame.init()
-        self.screen = pygame.display.set_mode((
-            settings.SCREEN_WIDTH,
-            settings.SCREEN_HEIGHT,
-        ))
         pygame.display.set_caption('PyDew')
+
+        # variables 
+        self.gamestate = "main menu"
```

Comment:
I suggest using IntEnums instead of strings, because it keeps the code consistent. And you will encounter no typo problem if you use enum constants instead of strings

Reply:
I don't quite understand the benefit of using enums instead of strings. Sure you have to make sure you spell the variable correctly with strings but you already have to do that with any variable name. Enums on the other hand require another library to be imported, have to define every option to a number, and then require a call to the variable like variablename.value. strings definitions are a single line and infinitely expandable without requiring any additional libraries or multiple lines of numbering. I've been programming for years and this is the first time I've come across enums. I had to read up on them and still don't quite understand the use. if we are wanting to keep the code simple for beginners, I'm not sure enums are the simplest way.

Reply:
Enums are constant names, so if you use completion helpers like in PyCharm for example you can easily select a constant between all available ones for an enum. Also, strings are typo-prone, so if for example your code set `self.gamestate` to `"main myenu"` somehow due to a typo, the game would break, since nothing would be planned for a value of `"main myenu"` as a separate game state. And it would be extremely painful to check where the typo occurred, as you might not even get an error for that. If you use enums and end up making a typo, the interpreter will actively signal it to you by raising a `NameError`, and the game will stop. Moreover, the resulting traceback will show you the exact line where the typo was